cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/MinTopicLeadersPerBrokerGoal.java

```diff
@@ -201,11 +203,12 @@ private void validateEnoughLeaderToDistribute(ClusterModel clusterModel, Optimiz
 
     for (Map.Entry<String, Integer> numLeadersPerTopic : numLeadersByTopicNames.entrySet()) {
       if (numLeadersPerTopic.getValue() < totalMinimumLeaderCount) {
+        ProvisionRecommendation recommendation = new ProvisionRecommendation.Builder(ProvisionStatus.UNDER_PROVISIONED)
+            .numPartitions(totalMinimumLeaderCount - numLeadersPerTopic.getValue()).topic(numLeadersPerTopic.getKey()).build();
         throw new OptimizationFailureException(
             String.format("[%s] Cannot distribute %d leaders over %d broker(s) with minimum required per broker leader count %d for topic %s.",
                           name(), numLeadersPerTopic.getValue(), eligibleBrokersForLeadership.size(), minTopicLeadersPerBroker(),
-                          numLeadersPerTopic.getKey()),
-            String.format("Increase the partition count of topic %s to at least %d.", numLeadersPerTopic.getKey(), totalMinimumLeaderCount));
```

Comment:
nit: I would like to keep this format `count of topic to at least %d`, but I admit that I can't come up with an elegant way to do so. 
Reason: To manually increase the partition count through CLI, operators give the count parameter as total count instead of how many more partitions to add. So displaying the total `totalMinimumLeaderCount` directly can help them avoiding the risk of miss calculating the addition of `numLeadersPerTopic` + `_numPartitions-to-add`.

Reply:
Please see my related [change request](https://github.com/linkedin/cruise-control/pull/1494#pullrequestreview-622401143) below.